fronts-client/src/components/form/ArticleMetaForm.tsx

```diff
@@ -1080,6 +1128,7 @@ const createMapStateToProps = () => {
 				article && selectFormFields(state, article.uuid, isSupporting),
 			kickerOptions: article ? selectArticleTag(state, cardId) : defaultObject,
 			imageSlideshowReplace: valueSelector(state, 'imageSlideshowReplace'),
+			showMainVideo: valueSelector(state, 'showMainVideo'),
```

Comment:
We need to access the Redux form value to bind to the checked status of the Select Video radio button. We already have the imageSlideshowReplace selector in the line above.